docs/GlusterFS Tools/gfind_missing_files.md

```diff
@@ -42,26 +42,27 @@ WARNING: Unable to convert some GFIDs to Paths, GFIDs logged to /root/test_resul
 Use bash gfid_to_path.sh <brick-path> /root/test_results/m3-4.txt_gfids to convert those GFIDs to Path
 Total Missing File Count : 126733
 ```
+
 In such cases, an additional step is needed to convert those gfids to paths.
 This can be used as shown below:
-```sh
- $bash gfid_to_path.sh <BRICK_PATH> <GFID_FILE>
+
+```console
+bash gfid_to_path.sh <BRICK_PATH> <GFID_FILE>
              BRICK_PATH - Full path of the brick.
              GFID_FILE  - OUTFILE_gfids got from gfind_missing_files.sh
 ```
-Things to keep in mind when running the tool
-============================================
+
+## Things to keep in mind when running the tool
+
 1. Running this tool can result in a crawl of the backend filesystem at each
    brick which can be intensive. To ensure there is no impact on ongoing I/O on
    RHS volumes, we recommend that this tool be run at a low I/O scheduling class
    (best-effort) and priority.
-```sh
```

Comment:
i dont understand the reason behind removing this

Reply:
Wrong indentation, triple backtick will not work under numbered or un-ordered lists. See the docs page rendered currently. Instead of number 2 it again started as number 1 because of this code block is breaking the list.

https://docs.gluster.org/en/latest/GlusterFS%20Tools/gfind_missing_files/

Reply:
nice catch